File: SoalA/product.java
public class product {
    /* product adalah parent dari semua class */

    /* atribut privat product */
    private String idProduct;
    private String price;

    /* membuat constructor */
    public product(String idProduct, String price) {
        this.idProduct = idProduct;
        this.price = price;
    }

    public product() {

    }

    /* Get-Set Methods idProduct */
    public void setID(String idProduct) {
        this.idProduct = idProduct;
    }

    public String getID() {
        return idProduct;
    }

    /* Get-Set Methods price */
    public void setPrice(String price) {
        this.price = price;
    }

    public String getPrice() {
        return price;
    }
}

File: SoalB/Pyhton/product.py
class product():
    # product adalah parent dari semua class

    # atribut private
    __idProduct = ""
    __price = ""

    # membuat constructor
    def __init__(self):
        self.__idProduct = ""
        self.__price = ""

    # get-set methods idProduct
    def setID(self, idProduct):
        self.__idProduct = idProduct

    def getID(self):
        return self.__idProduct

    # get-set methods price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
